### PR TITLE
E_CLASSROOM-267 [Bugfix] [Student details] avatar in the student deta…

### PR DESCRIPTION
**Pull request by CaneteJoash**
### Issue Link
https://framgiaph.backlog.com/view/E_CLASSROOM-267
### Definition of Done
- [X] When the user visit other user/ View student details, it should show her/his avatar.
- [X] Should display the proper avatar of the student

### Commands to Run
N/A

#### Main note
N/A
#### Pages Affected
- http://localhost:3003/students/id

### Scenarios/ Test Cases
- [X] When the user visit other user/ View student details, it should show her/his avatar.
- [X] Should display the proper avatar of the student
#### User Interface
N/A
#### User Experience 
N/A
#### Functionality
N/A
### Screenshots
image

### CODE REVIEW

File: src/pages/student/students/StudentDetail/index.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ import React, { useEffect, useState } from 'react';
 import { useParams } from 'react-router-dom';
 import { useToast } from '../../../../hooks/useToast';
 import style from './index.module.css';
-import { BiUser } from 'react-icons/bi';
+// import { BiUser } from 'react-icons/bi';
```
Comment (karlolee50) [inline src/pages/student/students/StudentDetail/index.js]: remove if not used

File: src/pages/student/students/StudentDetail/index.js [hunk 2]
```diff
@@ -102,8 +104,16 @@ const StudentDetail = () => {
...
+          <div>
+            {studentDetails?.avatar ? (
+              <div>
+                <img src={avatar} className={style.biUserPosition} />
+              </div>
+            ) : (
+              <div>
+                <img className={style.biUserPosition1} src="https://www.pngall.com/wp-content/uploads/5/Profile-Avatar-PNG.png"/>
+              </div>
+            )}
```
Comment (karlolee50) [inline src/pages/student/students/StudentDetail/index.js]: maybe we can add the ternary in the src and classname instead?
```suggestion
          <div>
            <div>
                <img src={studentDetails?.avatar ? studentDetails?.avatar : 'https://www.pngall.com/wp-content/uploads/5/Profile-Avatar-PNG.png'} className={tudentDetails?.avatar ? style.biUserPosition : style.biUserPosition1} />
              </div>
```

**Review by karlolee50 — CHANGES_REQUESTED** [review]
Review time: 10 minutes
Comment/s: 2

**Comment by catampongan** [comment]
## Approved PR #95
- [x] Able to display default avatar
- [x] Able to display profile picture of other students
----------
Screenshot 2022-02-09 163942
Screenshot 2022-02-09 165057
----------
- Total Test Time: 0.5 hr
- Total Iterations: 0
- Total Comments: 0

**Review by karlolee50 — APPROVED** [review]

**Pull request merged**